AAI/FAQ.md

```diff
@@ -4,21 +4,235 @@ title: AAI FAQ
+<hr style="border: 2px solid; margin: 2em auto;"/>    
 
-It is probable that a special token endpoint will exist in a future
-version of this profile that should prevent the `/userinfo` endpoint
-from being overloaded.
+## Flows
+
+The following sequence diagrams are included to help explain the intended flows
+documented in the accompanying specification. 
+
+### What is the complete end to end flow using `/userinfo`?
+
+(last updated June 2022)
+
+The flow as used by Elixir uses the initial *Passport-Scoped Access Token* as
+a token handed to downstream resource servers. These servers can use this token, in conjunction
+with a callback to the broker's *Userinfo Endpoint*, to obtain the *Passport* content in
+JSON format.
+
+{% plantuml %}
+
+hide footbox
+skinparam BoxPadding 10
+skinparam ParticipantPadding 20
+
+box "Researcher"  #eee
+actor       "User Agent"                as user
+participant Client                      as client
+end box
+
+box "AAI"
+participant Broker                      as broker
+collections "IdP"                       as idps
+end box
+
+box "Data Owner"
+collections "Visa Issuer(s)"            as issuers
+end box
+
+box "Data Holder"
+participant "Clearing House"            as clearing
+participant "Data"                      as data
+end box
+
+==OIDC==
+
+ref over user, client, broker, idps
+OIDC flow results in the client holding a *Passport-Scoped Access Token*.
+end ref
+
+==Use==
+
+client -> clearing : Client requests data
+note right
+{
+  Authorization: Bearer <Passport-Scoped Access Token>
+}
+end note
+
+clearing -> broker : Clearing house asks for passport content
+note right
+{
+  Authorization: Bearer <Passport-Scoped Access Token>
+}
+end note
+
+group#DeepSkyBlue #LightSkyBlue Informative Only (not defined in the specification)
+broker -> issuers : Fetch signed visa(s) for user
+broker <- issuers : Return signed visa(s) for user
+note right
+[
+  "<visa1>",
+  "<visa2>"
+]
+end note
+end
+
+clearing <- broker : UserInfo Endpoint returns passport content (plain JSON, not signed JWT)
+note right
+{
+  "iss": "https://<issuer-website>/",
+  "sub": "<subject-identifier>",
+  "jti": "<token-identifier>",
+  "ga4gh_passport_v1": [
+    "<visa1>",
+    "<visa2>",
+   ...
+  ]
+}
+end note
+
+
+note over clearing, data  #FFCCCB
+Decision is made to release data using information contained in the
+passport - and this decision is coordinated with the data system to
+facilitate that actual data release.
+end note
+
+client <- clearing : Client is given data
+
+
+{% endplantuml %}
+
+<hr style="width: 10em; margin: 2em auto;"/>   
+
+### What is the complete end to end flow using token exchange?
+
+(last updated June 2022)
+
+The exchange flow does not ever distribute the initial *Passport-Scoped Access Token* beyond
+the client application. A token exchange operation is executed by the client, in
+exchange for a *Passport Token* that may be used downstream to access resources. In this example flow, the
```

Comment:
Passport Token is not a defined term, the 1.2-draft-main branch defines [Passport](https://ga4gh.github.io/data-security/1.2-draft-main/aai-openid-connect-profile#term-passport) as "A signed and verifiable JWT container for holding Visas."
```suggestion
exchange for a *Passport* that may be used downstream to access resources. In this example flow, the
```